### ISSUE TITLE
[BUG] Source file comments breaks CMS templates

### ISSUE THREAD

**Issue by a2nt** [silverstripe/silverstripe-cms#2164]
```
SilverStripe\View\SSViewer:
  source_file_comments: true
```

![screenshot from 2018-05-14 10-09-36](https://user-images.githubusercontent.com/672794/39976618-f2fb760a-575e-11e8-988c-1c91fcf96c64.png)

**Comment by robbieaverill** [comment]
Yeah, this is an unfortunate side-effect of using `<% include AriaAttributes %>` as inline HTML attributes.

This only affects dev mode (devs shouldn't be using this setting outside of it).

**Comment by dhensby** [comment]
This is kind of a duplicate of https://github.com/silverstripe/silverstripe-framework/issues/6857
